newIDE/app/src/EventsBasedObjectEditor/EventBasedObjectChildrenEditor.js

```diff
@@ -162,16 +162,15 @@ export default class EventBasedObjectChildrenEditor extends React.Component<
     }
   };
 
-  _onObjectSelected = (selectedObjectName: string) => {
-    if (!selectedObjectName) {
-      this.setState({
-        selectedObjectNames: [],
-      });
-    } else {
-      this.setState({
-        selectedObjectNames: [selectedObjectName],
-      });
+  _onObjectSelected = (objectWithContext: ?ObjectWithContext = null) => {
+    const selectedObjectNames = [];
+    if (objectWithContext) {
+      selectedObjectNames.push(objectWithContext.object.getName());
```

Comment:
is that a good thing to keep the previous selection when selecting 1 element?
If that's on purpose, should the function be renamed to `_onAdditionalObjectSelected` ?

Reply:
EDIT: I've re-read the code and now understands this is the same :)